x-pack/platform/plugins/shared/ml/public/application/services/ml_api_service/saved_objects.ts

```diff
@@ -56,11 +57,11 @@ export const savedObjectsApiProvider = (httpService: HttpService) => ({
       version: '1',
     });
   },
-  syncSavedObjects(simulate: boolean = false) {
+  syncSavedObjects(simulate: boolean = false, addToAllSpaces?: boolean) {
     return httpService.http<SyncSavedObjectResponse>({
       path: `${ML_EXTERNAL_BASE_PATH}/saved_objects/sync`,
       method: 'GET',
-      query: { simulate },
+      query: { simulate, addToAllSpaces },
```

Comment:
would it be more flexible to have the `syncToSpace` parameter with `_all`, `default`, etc values?